tests/graphql/test-modify.graphql

```diff
@@ -8,7 +8,7 @@ input Foo {
 }
 
 type Query {
-  foo(input: Foo): String @modify(name: "data")
+  foo(input: Foo): String @modify(name: "data", omit: false)
```

Comment:
had to change this part in test file as conversion `SDL -> config -> SDL` now may differ due to fact that defaults are now adding to the config

Reply:
so the default is `true`?

Reply:
no, default is false, but in this test it fails comparing input `@modify(name: "data")` to the output `@modify(name: "data", omit: false)`